contao/controllers/BackendInstall.php

```diff
@@ -731,16 +728,9 @@ protected function enableSafeMode()
 
 		if (!\Config::get('coreOnlyMode'))
 		{
-			global $kernel;
-
-			$modules = array();
-
-			foreach ($kernel->getContaoBundles() as $bundle)
-			{
-				$modules[] = $bundle->getName();
-			}
+			$modules = \ModuleLoader::getActive();
 
-			if (count(array_diff($modules, array('ContaoCoreBundle', 'calendar', 'comments', 'devtools', 'faq', 'listing', 'news', 'newsletter', 'repository'))) > 0)
+			if (count(array_diff($modules, array('core', 'calendar', 'comments', 'faq', 'listing', 'news', 'newsletter'))) > 0)
```

Comment:
Nope, I basically reverted that to the old code, because `ModuleLoader` will map legacy module names (see https://github.com/contao/core-bundle/pull/55/files#diff-3c973c84d5ed12dfb4bee28f58f15e66R35)